src/synapse/azext_synapse/_params.py

```diff
@@ -3,34 +3,30 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 # pylint: disable=line-too-long
+from argcomplete import FilesCompleter
+from azext_synapse.constant import SparkBatchLanguage, SparkStatementLanguage
 from azure.cli.core.commands.parameters import name_type, tags_type, get_three_state_flag, get_enum_type, \
     resource_group_name_type
+from azure.cli.core.util import get_json_object
+from ._validators import validate_storage_account
 
 
 # pylint: disable=too-many-statements
 def load_arguments(self, _):
     # synapse spark
-
-    for scope in ['batch', 'session', 'session-statement']:
+    for scope in ['batch', 'session', 'statement']:
         with self.argument_context('synapse spark ' + scope) as c:
             c.argument('workspace_name', help='The name of the workspace.')
             c.argument('spark_pool_name', help='The name of the spark pool.')
 
     for scope in ['synapse spark batch', 'synapse spark session']:
         with self.argument_context(scope + ' create') as c:
             c.argument('job_name', arg_type=name_type, help='The spark batch or session job name.')

```

Comment:
```suggestion
            c.argument('job_name', arg_type=name_type, help='The spark batch job name.')
```

Reply:
this parameter is needed for `synapse spark session`. It is weird that when you run `synapse spark session create -h` it returns "The Spark batch job name."

---
In reply to: [418068442](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418068442) [](ancestors = 418068442)

Reply:
fixed. split with spark session

---
In reply to: [421220997](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r421220997) [](ancestors = 421220997,418068442)